frontend/appflowy_flutter/lib/shared/markdown_to_document.dart

```diff
@@ -28,3 +36,67 @@ String customDocumentToMarkdown(Document document) {
     ],
   );
 }
+
+Future<String> documentToMarkdownFiles(
+  Document document,
+  String path, {
+  AsyncValueSetter<Archive>? onArchive,
+}) async {
+  final List<Future<ArchiveFile>> fileFutures = [];
+
+  /// create root Archive and directory
+  final id = document.root.id,
+      archive = Archive(),
+      resourceDir = ArchiveFile('$id/', 0, null)..isFile = false,
+      fileName = p.basenameWithoutExtension(path),
+      dirName = resourceDir.name;
+
+  final markdown = documentToMarkdown(
```

Comment:
Please add the parsers into `customDocumentToMarkdown` instead of using `documentToMarkdown` directly.